addons/mail/static/src/models/composer/composer.js

```diff
@@ -717,6 +782,76 @@ function factory(dependencies) {
             return undefined;
         }
 
+        /**
+         * Post a message from the queue.
+         * @private
+         */
+        async _postNextMessageInQueued(message) {
```

Comment:
`_sendNextPendingMessage`, and the method should not receive the next message in param but read it from the models.